### PR TITLE
added chip icon and wand, moved main askAI page

### PR DESCRIPTION
**Pull request by PavelLinearB**

### CODE REVIEW

**Review by MishaKav — APPROVED** [review]
👍

**Review by vim-zz — APPROVED** [review]

File: docs/downloads/automation-library/integrations/askAI/askAI_improve.cm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    if:
+      - {{ pr.labels | match(term="askai tests") | some }}
+    run:
+      - action: add-comment@v1
+        args:
+          comment: |
+            # 🧪 Test Suggestions 🧪
+            {{ source | askAI("QA tester",
+              "Review the code diff and suggest improvements to enhance readability, performance, and maintainability. 
+              Focus only on the modified code and provide specific, actionable recommendations.", env.OPEN_AI_TOKEN) | encode }}
```
Comment (MishaKav) [inline docs/downloads/automation-library/integrations/askAI/askAI_improve.cm:20]: please remove the `encode` in all async functions, like:

```suggestion
              Focus only on the modified code and provide specific, actionable recommendations.", env.OPEN_AI_TOKEN) }}
```

File: docs/downloads/automation-library/integrations/askAI/askAI_document.cm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      - {{ pr.labels | match(term="askai tests") | some }}
+    run:
+      - action: add-comment@v1
+        args:
+          comment: |
+            # 🧪 Test Suggestions 🧪
+            {{ source | askAI("Technical writer",
+              "Review the code diff and generate documentation for any newly added features or significant changes. 
+              Focus on describing functionality, parameters, and expected behavior. 
+              Format the output in clear, concise bullet points or paragraphs.", env.OPEN_AI_TOKEN) | encode }}
```
Comment (MishaKav) [inline docs/downloads/automation-library/integrations/askAI/askAI_document.cm:21]:
```suggestion
              Format the output in clear, concise bullet points or paragraphs.", env.OPEN_AI_TOKEN) }}
```

**Review by MishaKav — CHANGES_REQUESTED** [review]

File: docs/downloads/automation-library/integrations/askAI/askAI_CR.cm [hunk 3]
```diff
@@ -7,7 +7,7 @@ triggers:
     - pr_created
 
 automations:
-  ask_ai_describe:
+  ask_ai_review:
```
Comment (MishaKav) [inline docs/downloads/automation-library/integrations/askAI/askAI_CR.cm:10]: remove the `encode`

File: docs/downloads/automation-library/integrations/askAI/askAI_tests.cm [hunk 4]
```diff
@@ -7,7 +7,7 @@ triggers:
     - pr_created
 
 automations:
-  ask_ai_describe:
+  ask_ai_tests:
```
Comment (MishaKav) [inline docs/downloads/automation-library/integrations/askAI/askAI_tests.cm:10]: remove the `encode`

File: docs/downloads/automation-library/integrations/askAI/askAI_summarize.cm [hunk 5]
```diff
@@ -7,7 +7,7 @@ triggers:
     - pr_created
 
 automations:
-  ask_ai_describe:
+  ask_ai_summarize:
```
Comment (MishaKav) [inline docs/downloads/automation-library/integrations/askAI/askAI_summarize.cm:10]: remove the `encode`

**Review by MishaKav — CHANGES_REQUESTED** [review]

**Review by MishaKav — APPROVED** [review]
Lg

**Review by MishaKav — APPROVED** [review]
Looks good.

In general, I don't like the `# 🧪 Test Suggestions 🧪`, because it make the title too big.
I think we may change it to something more elegant. Bold or ###

**Pull request merged**